lib/modules/datasource/galaxy-collection/index.ts

```diff
@@ -119,9 +119,9 @@ export class GalaxyCollectionDatasource extends Datasource {
   }
 
   @cache({
-    namespace: `datasource-${GalaxyCollectionDatasource.id}-detailed-version`,
+    namespace: `datasource-${GalaxyCollectionDatasource.id}`,
     key: (_packageName: string, _versionsUrl: string, basicRelease: Release) =>
-      basicRelease.version,
+      `getVersionDetails:${basicRelease.version}`,
```

Comment:
it seems we should add versions url to the key for more uniqueness

Reply:
Yes, this one always looked strange to me